test/modules/core/lib/pick-layers.spec.js

```diff
@@ -157,7 +157,7 @@ const TEST_CASES = [
             height: 100
           },
           results: {
-            count: 33
+            count: 34
```

Comment:
Has a breaking change been introduced?

Reply:
As commented above, this change required to updates to deviceRect `width` and `height` calculations.